extensions/wikia/UserRenameTool/UserRenameToolProcess.class.php

```diff
@@ -638,20 +768,26 @@ public function updateLocal() {
 
```

Comment:
Possibly outside the scope of this PR, but refactoring huge methods like this into smaller work specific methods would help readability greatly.